cmd/riverui/logger.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"log/slog"
+	"os"
+	"strings"
+)
+
+var logger *slog.Logger //nolint:gochecknoglobals
+
+func initLogger() {
+	options := &slog.HandlerOptions{Level: getLogLevel()}
+	logger = slog.New(slog.NewTextHandler(os.Stdout, options))
+}
+
+func getLogLevel() slog.Level {
+	if isDebug() {
+		return slog.LevelDebug
+	}
+
+	env := strings.ToLower(os.Getenv("RIVER_LOG_LEVEL"))
+
+	switch env {
+	case "debug":
+		return slog.LevelDebug
+	case "warn":
+		return slog.LevelWarn
+	case "error":
+		return slog.LevelError
+	}
+
+	return slog.LevelInfo
```

Comment:
cleaner to keep this as a `default` IMO:

```suggestion
	case "error":
		return slog.LevelError
	default:
		return slog.LevelInfo
	}
```